lib/DiscoveryStreamFeed.jsm

```diff
@@ -442,7 +467,13 @@ this.DiscoveryStreamFeed = class DiscoveryStreamFeed {
 
   filterBlocked(data, type) {
     if (data && data[type] && data[type].length) {
-      const filteredItems = data[type].filter(item => !NewTabUtils.blockedLinks.isBlocked({"url": item.url}));
+      const filteredItems = data[type].filter(item => {
```

Comment:
I believe filter can happen here for spocs or recs. Do you need to do anything to handle that too?

Reply:
Good point. Yep, we should handle this.